mmv1/products/firebasedatabase/api.yaml

```diff
@@ -0,0 +1,91 @@
+objects:
+  - !ruby/object:Api::Resource
+    name: 'Instance'
+    min_version: beta
+    base_url: projects/{{project}}/locations/{{region}}/instances/
+    self_link: projects/{{project}}/locations/{{region}}/instances/{{instance_id}}
+    create_url: projects/{{project}}/locations/{{region}}/instances?databaseId={{instance_id}}
+    description: A Firebase Realtime Database instance.
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation':
+          'https://firebase.google.com/products/realtime-database'
+      api: 'https://firebase.google.com/docs/reference/rest/database/database-management/rest'
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: 'region'
+        description: |
+          A reference to the region where the Firebase Realtime database resides.
+        required: true
+        input: true
+        url_param_only: true
+      - !ruby/object:Api::Type::String
+        name: 'instance_id'
+        description: |
+          The globally unique identifier of the Firebase Realtime Database instance. 
+          Instance IDs cannot be reused after deletion.
+        required: true
+        input: true
+        url_param_only: true
+    properties:
+      - !ruby/object:Api::Type::String
+        name: name
+        description: |
+          The fully-qualified resource name of the Firebase Realtime Database, in the
+          format: projects/PROJECT_NUMBER/locations/REGION_IDENTIFIER/instances/INSTANCE_ID
+          PROJECT_NUMBER: The Firebase project's [`ProjectNumber`](https://firebase.google.com/docs/projects/api/reference/rest/v1beta1/projects#FirebaseProject.FIELDS.project_number)
+          Learn more about using project identifiers in Google's [AIP 2510 standard](https://google.aip.dev/cloud/2510).
+        output: true
+      - !ruby/object:Api::Type::String
+        name: database_url
+        description: |
+          The database URL in the form of https://{instance-id}.firebasio.com for us-central1 instances
+          or https://{instance-id}.{region}.firebasedatabase.app in other regions.
+        output: true
+      - !ruby/object:Api::Type::Enum
+        name: type
+        description: |
+          The database type.
+          Each project can create one default Firebase Realtime Database, which cannot be deleted once created.
+          Creating user Databases is only available for projects on the Blaze plan.
+          Projects can be upgraded using the Cloud Billing API https://cloud.google.com/billing/reference/rest/v1/projects/updateBillingInfo.
+        input: true
+        values:
+          - :DEFAULT_DATABASE
+          - :USER_DATABASE
+        default_value: :USER_DATABASE
+      - !ruby/object:Api::Type::Enum
+        name: state
+        description: |
+          The current database state. Set desired_state to disable or reenable the database.
```

Comment:
nit: "Set desired_state to :DISABLED to disable the database and :ACTIVE to reenable the database"

Reply:
Done